specification/healthbot/resource-manager/Microsoft.HealthBot/stable/2020-10-20/healthbot.json

```diff
@@ -0,0 +1,630 @@
+          "Get Operations": {
+            "$ref": "./examples/GetOperations.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK.",
+            "schema": {
+              "$ref": "#/definitions/AvailableOperations"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "CheckNameAvailabilityRequestBody": {
+      "description": "The request body for a request to to check availability of a bot name.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "the name of the bot for which availability needs to be checked.",
+          "type": "string"
+        },
+        "type": {
+          "description": "the type of the bot for which availability needs to be checked",
+          "type": "string"
+        }
+      }
+    },
+    "CheckNameAvailabilityResponseBody": {
+      "description": "The response body returned for a request to check availability of a bot name.",
+      "type": "object",
+      "properties": {
+        "nameAvailable": {
+          "description": "indicates if the bot name is valid and available.",
+          "type": "boolean"
+        },
+        "reason": {
+          "description": "Reason for the result",
+          "type": "string"
+        },
+        "message": {
+          "description": "additional message from server",
+          "type": "string"
+        }
+      }
+    },
+    "Resource": {
+      "description": "Azure resource",
+      "x-ms-azure-resource": true,
+      "properties": {
+        "id": {
+          "description": "Specifies the resource ID.",
+          "type": "string",
+          "readOnly": true
+        },
+        "name": {
+          "description": "Specifies the name of the resource.",
+          "type": "string",
+          "readOnly": true
+        },
+        "location": {
+          "description": "Specifies the location of the resource.",
+          "type": "string"
+        },
+        "type": {
+          "description": "Specifies the type of the resource.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "Bot": {
+      "type": "object",
+      "description": "Bot resource definition",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/BotProperties",
+          "description": "The set of properties specific to healthcare bot resource."
+        }
+      }
+    },
+    "BotProperties": {
```

Comment:
Can you briefly describe what the user does with a Bot resource once its created?  How do they get value out of it?

Reply:
Yes, I added a short description of what a HealthBot is, and what value it provides for the users.